server/tables/tables.go

```diff
@@ -158,6 +158,9 @@ type Group struct {
 
 	SharingEnabled bool `gorm:"default:true"`
 
+	// If enabled, this group can use their own executors instead of the installation-wide shared executors.
```

Comment:
is "can" what you mean here? Does this bool being set force them to use their own or no?

Reply:
changed "can use" to "will always use"